pages/linux/mashtree.md

```diff
@@ -0,0 +1,17 @@
+# mashtree
+
+> Makes a fast tree from genomes.
+> Does not make a phylogeny.
+> More information: <https://github.com/lskatz/mashtree>.
+
+- Fastest method to create a tree from fastq and/or fasta files using multiple threads, piping into a DND file:
```

Comment:
Fastest in comparison to what? We try to avoid subjective descriptions, we try to focus on describe what the example do.

Reply:
I tried to fix it so that it says "within mashtree."  This software has a faster method and a more accurate method, as compared to each other.